lib/factory_girl_rails/railtie.rb

```diff
@@ -10,11 +10,10 @@ class Railtie < Rails::Railtie
     end
 
     initializer "factory_girl.set_factory_paths" do
-      FactoryGirl.definition_file_paths = [
-        Rails.root.join('factories'),
-        Rails.root.join('test', 'factories'),
-        Rails.root.join('spec', 'factories')
-      ]
+      paths = FactoryGirl.definition_file_paths || []
```

Comment:
I like the general idea here, but I don't know if this would totally solve the issue of allowing custom definition_file_paths. I think the initializer will run before any of your application code, so you wouldn't have a chance to customize these paths early enough to have an effect. We could maybe add a configuration option `config.factory_bot.definition_file_paths` to solve this.

I know it has been a while. Are you interested in working on this? I would be happy to help, or I can take over if you are no longer interested. If I don't hear from you I will look into this next week.